### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by jackdyson31** [michaeljklein/colorblind-lcd#1]
Hi Michael,

Just like to say thanks for doing this, it really helped me. Couldn't have been easy making that thing (looking at the VCGT curves). I was having major issues with purples and what not popping out of my high definition LCD  - made it unusable until I worked out what was at the root of it and found your profile.

Cheers again,

Jack

PS Could you tell me how (steps) you made this profile please so I can sort myself out for one as well ?

**Comment by michaeljklein** [comment]
Hi Jack,

Glad to hear it helped!

Sure. I don't have the exact steps, so here's a summary of the process I used:

At the time, I was pretty bothered by the possibility that my color blindness was making me miss
out on finer details of images.


## Research

I looked up what key words and phrases that were relevant to [Red-green Color Blindness](https://en.wikipedia.org/wiki/Color_blindness#Red–green_color_blindness) 
and found a bunch of these images: https://en.wikipedia.org/wiki/Ishihara_test

I looked through them, and picked out a collection of maybe a dozen that were "on the edge", where I could just barely make out the outlines of the numbers.

I prepared a random sampling of other images with more/less detail/color. (I don't remember exactly what I did, probably just used [Google Images](https://images.google.com) with some random phrases and picked some that I liked).


## Picking the configuration

I went to "Displays" in "System Preferences", then "Color", "Calibrate".

I followed the steps and messed with a bunch of the settings, using the images as a guide.
In other words, I'd tweak a setting, check a couple images for comparison, then try something else.

I tried to be pretty exhaustive with my tweaking and worked on it until I felt comfortable with the results.


Cheers,
Michael

**Comment by jackdyson31** [comment]
Thanks a lot for doing it Michael, I really appreciate it.

Just a quick question - what software did you use for creating the icc
itself? I want to mess around with the gamma curves you made and send up a
new LUT ...

Best

Jack

**Comment by michaeljklein** [comment]
The `.icc` file is the color-profile configuration file _generated by OSX System Preferences_, which you can find:

```
http://www.colourmanagement.net/advice/where-are-my-icc-profiles

7 and later: Windows\system32\spool\drivers\color
XP: Windows\system32\spool\drivers\color
2000+ME: WinNT\system\spool\drivers\color

mac up to OS 9.x Colorsync Profiles folder inside the System folder.
windows NT4: WinNT\System32\Color
windows 95-98: Windows\System\Color
```


In other words, I did not use any other software to create the `.icc`, other than OSX's System Preferences.


If you look in the `.icc` file (skipping some unprintable characters):

```
\applmntrRGB
XYZ
#acspAPPL
appl
desc
Ptdscm
aarg
vcgt
ndin
>chad,mmod)4(bTRC
gTRC
aabg
aagg
desc
ColorBlind LCD Calibrated
mluc
enUS2
ColorBlind LCD Calibrated
text
Copyright Apple Inc., 2014
%+28>ELRY`gnu|```
```

You can see that it appears to have some sort of "watermark."
I am not a lawyer, but I believe the copyright _might_ be on the file's format,
i.e. it may be Apple's proprietary version of an ICC specification: https://en.wikipedia.org/wiki/ICC_profile#References_in_standards
